### ISSUE TITLE
Add a font selection to UI

### ISSUE THREAD

**Issue by epic6666** [ksnip/kImageAnnotator#130]
- Current behavior

In order to change the font, one has to go into Options -> Settings -> Annotator and change it in there.

- Requested behavior

That feature is removed from settings and instead an entirely new user interface for configuring fonts and font color is created. (See the ShareX text tool window) Basically, it allows you to preview how a particular font (bold, italic or underlined) + color combo looks. Pressing OK would set the text tool to use that last chosen combination.

**Comment by medmedin2014** [comment]
We can't access annotator settings from Spectacle.

**Comment by DamirPorobic** [comment]
We are not developing Spectacle, how Spectacle handles this stuff is up to the KDE folks. 

Anyways, this settings will probably get into the UI soon.

**Comment by medmedin2014** [comment]
I already opened a bug at KDE and it was resolved upstream linked to this project, so I will reopen it and see their reply :)

**Comment by DamirPorobic** [comment]
> Anyways, this settings will probably get into the UI soon.

This will probably happen before the KDE guys do something. Just saying.

**Comment by DamirPorobic** [comment]
This is now implemented, you can test with ksnip lates build.

**Comment by medmedin2014** [comment]
If you create a text then change its font multiple times, then when you click first inside font combobox (not on arrow but inside the component) then you click on combobox arrow ksnip crashes.
```

Ksnip: 1.9.0-continuous
Operating System: Manjaro Linux
KDE Plasma Version: 5.21.4
KDE Frameworks Version: 5.81.0
Qt Version: 5.15.2
Kernel Version: 5.4.112-1-MANJARO
OS Type: 64-bit
Graphics Platform: X11
```

https://user-images.githubusercontent.com/7027319/115969619-45514f00-a52d-11eb-8651-95d023ebfd10.mp4

**Comment by DamirPorobic** [comment]
I cannot reproduce that on my machine, I tried to do what you do there but no crash. How do you install ksnip? Is this the appImage?

**Comment by medmedin2014** [comment]
Yes it's an AppImage. I can't install the latest version on Manjaro, the current version available on AUR is 1.8.2-2.
